docs/stdlib/objects.rst

```diff
@@ -14,20 +14,18 @@ Base Objects
       - Root for user-defined object types
 
 
-``std::BaseObject`` is the root of the object type hierarchy and all
-object types in EdgeDB, including system types, extend ``std::BaseObject``
-directly or indirectly.  User-defined object types extend from ``std::Object``,
-which is a subtype of ``std::BaseObject``.
+:eql:type:`BaseObject` is the root of the object type hierarchy and all
+object types in EdgeDB, including system types, whether extending it directly
+or indirectly. User-defined object types extend from the :eql:type:`Object`
+type, which is a subtype of ``std::BaseObject``.
 
 
 ---------
 
 
 .. eql:type:: std::BaseObject
 
-    Root object type.
-
-    Definition:
+    Represents the root object of a type:
```

Comment:
```suggestion
    The root object type.
```
The rewrite changes the meaning. ``BaseObject`` doesn't represent anything — it is a type — and it's not the root object _of_ a type — it is the root object type, meaning the root type of all objects.